webapp/src/app.tsx

```diff
@@ -3859,6 +3978,10 @@ export class ProjectView
         return this.state.tutorialOptions != undefined;
     }
 
+    onTutorialLoaded() {
```

Comment:
"tutorial.editorLoaded" tick event got dropped?

Reply:
I figured that as long as I was piping tutorial events I might as well make it explicit and add a new event

Reply:
oh i mean, do we still want it for telemetry reasons?

Reply:
oh yeah sure i'll put it back